api/src/main/java/org/openmrs/module/biometric/api/service/ParticipantService.java

```diff
@@ -143,4 +141,11 @@ List<PatientResponse> findPatientsByUuids(Set<String> uuids)
    */
   List<SyncTemplateResponse> getBiometricDataByParticipantIds(Set<String> uuids);
 
+  /**
+   * Retrieves all patient's identifiers of given type
+   *
+   * @param identifierName identifier type name
+   * @return list of patient identifiers
+   */
+  List<PatientIdentifier> getAllIdentifiersByName(String identifierName);

```

Comment:
LGTM Thanks @druchniewicz . How about making the title as _**Fetch All patient identifier values**_   .